concepts/regular-expressions/introduction.md

```diff
@@ -0,0 +1,188 @@
+# Introduction
```

Comment:
The introduction file for the concept and the exercise should be as short as possible. Think about what is really crucial to understand the basic concept and to solve the exercise and which parts can be removed/shortened.

(I would recommend you do this after you reconciled the about and introduction fixes you did.)

Reply:
Okay so I've tried shortening the introduction file for both the concept and the exercise, but I think most of the paragraphs are key to solve the challenge. Please let me know if there are some that we could get rid of, simplify or just change the way are explained 🤔